meta-lmp-base/recipes-support/fioconfig/fioconfig_git.bb

```diff
@@ -9,7 +9,7 @@ SRC_URI = "git://${GO_IMPORT} \
 	file://fioconfig.service \
 	file://fioconfig-extract.service \
 "
-SRCREV = "6d9f25f1c402eaa23bb98c0ce04355cd96e25647"
+SRCREV = "14512a50207a882522dcaa2e0c116f5a4da264f1"
```

Comment:
I just added another commit to master you may as well pull in:

 https://github.com/foundriesio/fioconfig/commit/1db6a4fdee49a69a64c4c0ac6f92bc2aa84c874f